modules/apps/object/object-test/src/testFunctional/tests/paths/ObjectAdmin.path

```diff
@@ -30,6 +30,26 @@
 		<td>//div[(@class='table-list-title') and contains (.,'${key_name}')]/../../div[contains(text(),'${key_system}')]</td>
 		<td></td>
 	</tr>
+		<tr>
+		<td>VIEW_CUSTOM_OBJECT</td>
+		<td>//div[contains(text(),'No')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>SELECT_FIRST_CUSTOM_OBJECT</td>
+		<td>//div[contains(text(),'No')]/..//button[@class='dropdown-toggle component-action dropdown-toggle ml-1 btn btn-monospaced btn-unstyled']</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>DELETE_CUSTOM_OBJECT</td>
+		<td>//button[(@class='dropdown-item') and contains(text(),'Delete')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>MESSAGE_SUCCESS</td>
+		<td>//div[@class='alert alert-dismissible alert-success']</td>
+		<td></td>
+	</tr>
```

Comment:
Since I've asked to remove the `VerifyElementPresent` call from the macro, you can remove it.

Reply:
@rodrigocunhaa Done.